api/command.go

```diff
@@ -87,17 +87,16 @@ func listCommands(c *Context, w http.ResponseWriter, r *http.Request) {
 }
 
 func executeCommand(c *Context, w http.ResponseWriter, r *http.Request) {
-	props := model.MapFromJson(r.Body)
-	command := strings.TrimSpace(props["command"])
-	channelId := strings.TrimSpace(props["channelId"])
+	post := model.PostFromJson(r.Body)
```

Comment:
If we add a new struct as I suggested above, we can also keep the `executeCommand` api backwards compatible since the `command` field won't have to be renamed to `message`.